src/TypeSpec.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -906,6 +906,30 @@ export function getUsages(
                     appendUsage(name, UsageFlags.Output);
                 }
             }
+            /* calculate the usage of the LRO result type. */
+            const metadata = getLroMetadata(program, op.operation);
+            if (metadata !== undefined) {
+                let bodyType: Model;
+                if (
+                    op.verb !== "delete" &&
+                    metadata.finalResult !== undefined &&
+                    metadata.finalResult !== "void"
+                ) {
+                    bodyType = metadata.finalEnvelopeResult as Model;
+                    if (bodyType) {
+                        getAllEffectedModels(
+                            bodyType,
+                            new Set<string>()
+                        ).forEach((element) => {
+                            logger.log(
```

Comment:
Could you help to explain why we need to add log here?  Any attention needed for the user?

Reply:
For debug purpose. I removed it.